index.bs

```diff
@@ -836,7 +849,7 @@ interface XRBoundedReferenceSpace : XRReferenceSpace {
 };    
 </pre>
 
-The origin of a {{XRBoundedReferenceSpace}} MUST be positioned at the floor, such that the `y` axis equals `0` at floor level. The `x` and `z` position and orientation are initialized based on the conventions of the underlying platform, typically expected to be near the center of the room facing in a logical forward direction.
+The [=native origin=] of a {{XRBoundedReferenceSpace}} MUST be positioned at the floor, such that the `y` axis equals `0` at floor level. The `x` and `z` position and orientation are initialized based on the conventions of the underlying platform, typically expected to be near the center of the room facing in a logical forward direction. It is allowed to drift over time.
```

Comment:
> It is allowed to drift over time.

Reason for this addition? One of the benefits of the bounded space should be that it's origin and bounds stay reasonably static relative to the environment (otherwise the bounds are pretty useless.) I understand that inside-out-tracked systems may introduce some minor inconsistencies to the origin over time, but in order to supply the data needed by a bounded reference space that should be viewed as errors for the system to correct and not acceptable drift that may continue to accumulate over time.

Reply:
Whoops, mixed up bounded and unbounded